static/monorail-2.0.yaml

```diff
@@ -650,6 +667,130 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+  /hooks:
+    get:
+      description: Get the RackHD hooks configuration properties.
```

Comment:
Suggest refactor the description "Get all hooks", the word "RackHD" and "configuration" is not necessary. The same for other description.